DiamondLightSource/web-ui-components
# [LIMS-1407] Add Input props to Flipper component

JIRA ticket: [LIMS-1407](https://jira.diamond.ac.uk/browse/LIMS-1407)

**Summary**:

Allows library users to use Input props with the Input portion of the Flipper component

**Changes**:

   - Allows library users to use Input props with the Input portion of the Flipper component
   - Remove maximum width from Flipper

**To test**:

    - Open Storybook, check that Flipper now takes up whole width of the container by default
